csrc/scheduler/vectorize_helper.cpp

```diff
@@ -365,9 +392,48 @@ std::vector<IterDomain*> ContiguousInnerDimensionsMapper::projectId(
     distributePE(merge_or_split);
   };
 
-  auto clear_left_of = [&frontier](IterDomain* id) {
-    auto it = std::find(frontier.begin(), frontier.end(), id);
-    if (it != frontier.end()) {
+  auto propagateResize = [&frontier, this](Resize* resize_op, bool p2c) {
+    // TODO: resize could affect the offset/pad_extent impacting vectorization
+    // factor on the loading inputs. But we shouldn't use that to block the
```

Comment:
Yes we are using that here. I'm just wondering if that is the right approach supporting slices, since it's going to be more restrictive.
I should remove this comment, since it's only adding confusion here.